sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -42,6 +42,14 @@ private[parquet] class ParquetFilters(pushDownDate: Boolean, pushDownStartWith:
   private val makeEq: PartialFunction[DataType, (String, Any) => FilterPredicate] = {
     case BooleanType =>
       (n: String, v: Any) => FilterApi.eq(booleanColumn(n), v.asInstanceOf[java.lang.Boolean])
+    case ByteType =>
```

Comment:
Let us update the comment in the line 34. 
```
Note: For now timestamp SQL type is not supported for pushing down, because the corresponding
      Parquet type can be Long or INT96. There is no context about Parquet type here.
```

Reply:
Probably `INT64 or INT96` (which is long vs binary) might be a bit better while we are here.